src/algebra/big_operators/basic.lean

```diff
@@ -1276,6 +1276,26 @@ end comm_group_with_zero
 
 end finset
 
+namespace fintype
+
+open finset
+
+/-- `fintype.prod_equiv` is a specialization of `finset.prod_bij` that
+automatically fills in most arguments. -/
+@[to_additive "`fintype.sum_equiv` is a specialization of `finset.sum_bij` that
+automatically fills in most arguments."]
+lemma prod_equiv {α β M : Type*} [fintype α] [fintype β] [comm_monoid M]
+  (e : α ≃ β) (f : α → M) (g : β → M) (h : ∀ x, f x = g (e x)) :
+  ∏ x : α, f x = ∏ x : β, g x :=
```

Comment:
I'm curious whether this is more or less useful as 
```suggestion
  (e : α ≃ β) (f : α → M) (g : β → M) :
  ∏ x : α, g (e x) = ∏ x : β, g x :=
```

Reply:
I'm still going through all the usages (mostly waiting for code to compile...), but it looks like there is only one instance where `h = refl` would work out of about 10. So I don't think the change is worth it.

Reply:
I guess this comes down to whether the lemma is intended to be used via apply or via `rw`.

Reply:
My request is stupid, the lemma I'm describing already exists and is called `equiv.prod_comp`!